File: Programming EF Code First/chapter6/Chapter6/Chapter 6 VB/Fluent/Model/Destination.vb
Namespace Model
  Public Class Destination
	Public Property DestinationId() As Integer
	Public Property Name() As String
	Public Property Country() As String
	Public Property Description() As String
	Public Property Photo() As Byte()
	Public Property TravelWarnings() As String
	Public Property ClimateInfo() As String

	Public Property Lodgings() As List(Of Lodging)

	Private _todayForecast As String
	Public Property TodayForecast() As String
	  Get
		  Return _todayForecast
	  End Get
	  Set(ByVal value As String)
		  _todayForecast = value
	  End Set
	End Property
  End Class
End Namespace

File: Programming EF Code First/chapter6/Chapter6/BAGA_FluentAPI/Model/Destination.cs
using System.Collections.Generic;

namespace Model
{
  public class Destination
  {
    public int DestinationId { get; set; }
    public string Name { get; set; }
    public string Country { get; set; }
    public string Description { get; set; }
    public byte[] Photo { get; set; }
    public string TravelWarnings { get; set; }
    public string ClimateInfo { get; set; }

    public List<Lodging> Lodgings { get; set; }

    private string _todayForecast;
    public string TodayForecast
    {
      get { return _todayForecast; }
      set { _todayForecast = value; }
    }
  }
}